lib/plugin/compilers/i18n.json.coffee

```diff
@@ -4,124 +4,147 @@ helpers = share.helpers
 compilers = share.compilers
 compiler_configuration = share.compiler_configuration
 
-compilers.i18n_json = (compileStep) ->
-  compiler_configuration.registerInputFile(compileStep)
```

Comment:
Ok that was my intuition, so I should merge the 3 defined compilers into one. Could you confirm that you get the whole list of matching files in `processFilesForTarget` even if only one file is changed?